policies/templates/gcp_resource_value_pattern_v1.yaml

```diff
@@ -98,31 +115,55 @@ spec:
        		"value": get_default_by_path(asset.resource.data, field_name, ""),
        	}
        
-       	not is_valid(rule_data)
+       	is_valid(mode, rule_data)
        
-       	message := sprintf("%v does not have a valid field named '%v' with a value '%v' matching pattern '%v'.", [
+       	message := sprintf("%v has %v violation for field named '%v' with a value '%v' matching pattern '%v'.", [
        		asset.name,
+       		mode,
        		field_name,
        		rule_data.value,
        		rule_data.pattern,
        	])
        
-       	metadata := {"resource": asset.name}
+       	metadata := {
+       		"resource": asset.name,
+       		"mode": mode,
+       	}
        }
        
        ###########################
        # Rule Utilities
        ###########################
        
-       is_valid({
+       is_valid(mode, {
```

Comment:
Why do you explicitly set the structure again here? Is it for readability?

If so, why "value" is an exception (an field_value is used in the call on line 146)?

Reply:
This was to keep the old format, but I've cleaned it up- let me know your thoughts!